data/refprog/refprog-xml-importer/src/main/java/com/farao_community/farao/data/refprog/refprog_xml_importer/RefProgImporter.java

```diff
@@ -47,23 +47,29 @@ public static ReferenceProgram importRefProg(InputStream inputStream, OffsetDate
         }
         List<ReferenceExchangeData> exchangeDataList = new ArrayList<>();
         document.getPublicationTimeSeries().forEach(timeSeries -> {
-            Country outArea = null;
-            Country inArea = null;
+            ReferenceProgramArea outArea;
+            String outAreaValue = timeSeries.getOutArea().getV();
             try {
-                outArea = (new EICode(timeSeries.getOutArea().getV())).getCountry();
+                new EICode(outAreaValue).getCountry();
+                outArea = new ReferenceProgramArea(outAreaValue);
             } catch (IllegalArgumentException e) {
                 LOGGER.warn("EIC code {} is not mapped to a country. The flow from this area will not be saved.", timeSeries.getOutArea().getV());
```

Comment:
same as previous remark :p

Reply:
Indeed, I have removed this useless try/catch block